hagrid/reservations/export_pdf.py

```diff
@@ -0,0 +1,342 @@
+from io import BytesIO
+
+from reportlab.pdfbase import pdfmetrics
+from reportlab.pdfgen import canvas
+from reportlab.platypus import Paragraph
+from reportlab.lib.pagesizes import A4
+from reportlab.lib.styles import ParagraphStyle
+from reportlab.lib.colors import black
+
+import datetime
+import logging
+import qrcode
+import time
+
+from .models import Reservation, ReservationPosition, ReservationPart
+from hagrid.products.models import Variation
+
+logger = logging.getLogger(__name__)
+
+QR_TEXT_FORMAT = "https://c3foc.net/reservations/{0}/"
```

Comment:
Please use django url reversing to build absolute URLs instead of hardcoding paths or even urls!
This should be possible, right!?

Reply:
Like I said. WIP. Before changing this I'd like to discuss what the QR code should actually do now since the new software doesn't implement a "set reservationstate to packed" API call.